qiskit/circuit/quantumcircuit.py

```diff
@@ -3260,6 +3269,8 @@ def draw(
         # pylint: disable=cyclic-import
         from qiskit.visualization import circuit_drawer
 
+        justify = justify if _is_valid_justify_arg(justify) else "left"
```

Comment:
```suggestion
        # We set this here to avoid the deprecation triggering again
        # from circuit_drawer. This code should be removed once
        # the deprecation is complete.
        justify = justify if _is_valid_justify_arg(justify) else "left"
```

Reply:
Actually, I added something on the lines, I guess a commit later, from your review 

I'll merge both now 👌